pymc/stats/log_density.py

```diff
@@ -169,9 +169,10 @@ def compute_log_density(
 
     n_pts = len(posterior_pts)
     logdens_dict = _DefaultTrace(n_pts)
-    indices = range(n_pts)
     if progressbar:
-        indices = progress_bar(indices, total=n_pts, display=progressbar)
+        indices = track(range(n_pts), description="Computing log density")
```

Comment:
Is there a reason this uses track vs the other places that use a progress bar?

Reply:
Its just a really simple use case (not sampling/fitting) so I kept it simple, but I can convert it just to keep things uniform.